oak-run/src/main/js/oak-mongo.js

```diff
@@ -329,6 +329,53 @@ var oak = (function(global){
         return {nRemoved : count};
     };
 
+    /**
+     * Finds all the descendants with a _modified timestamp older than the
+     * given timestamp.
+     *
+     * @memberof oak
+     * @method findDescendantsOlderThan
+     * @param {string} path of the parent node to search.
+     * @param {number} timestamp to compare with.
+     */
+    api.findDescendantsOlderThan = function(path, timestamp) {
+        var count = 0;
+        var depth = pathDepth(path);
+        var prefix = path + "/";
+        depth++;
+        var query = { _id: pathFilter(depth, prefix), _modified: {$lt: timestamp}};
+        db.nodes.find(query).forEach(function(doc) {
+            print(doc._id);
+            count++;
+        });
+        print("nCandidates : " + count);
+    }
+
+    /**
+     * Removes all the descendants with a _modified timestamp older than the
+     * given timestamp.
+     * Use removeDescendantsAndSelf() first to get the list of nodes to remove.
```

Comment:
```suggestion
     * Use findDescendantsOlderThan() first to get the list of nodes to remove.
```

Was this indeed meant to refer to removeDescendantsAndSelf, or not findDescendantsOlderThan?